GNUmakefile

```diff
@@ -1,38 +1,41 @@
-CC?=cc
-CFLAGS?=-O3 -flto -Wall -g -Werror=implicit-function-declaration -Werror=int-conversion
+CFLAGS?=-O3
 LDLIBS?=-lm
 INSTALL?=install
 PREFIX?=/usr
 
+# Skip platform.mk for `*clean*`
```

Comment:
No need to check if we aren't compiling.

Only for `clean`, since `install` *may* compile.

Reply:
Need to handle the pathological "clean all" so taking it out.